buildutils/torque.spec.in

```diff
@@ -61,9 +61,9 @@
 %define ac_trqauthd_sock_dir %{?trqauthd_sock_dir:%{trqauthd_sock_dir}}%{!?trqauthd_sock_dir:/tmp}
 
 ### Build Requirements
-%define breq_cpuset   %{?with_cpuset:hwloc-devel}
+%define breq_cpuset   %{!?hwloc_path:%{?with_cpuset:hwloc-devel}}
 %define breq_gui      %{?with_gui:tcl-devel tk-devel tclx}
-%define breq_numa     %{?with_cpuset:hwloc-devel}
+%define breq_numa     %{!?hwloc_path:%{?with_cpuset:hwloc-devel}}
 %define breq_munge    %{?with_munge:munge-devel}
```

Comment:
I'm still unsure of the wisdom of this change in particular.  It seems that it's trying to remove the build dependency (i.e., `BuildRequires:`) for the `hwloc-devel` package if a path is set for `hwloc`.  As a rule, if one is building and installing TORQUE via RPM, its dependencies should also be installed via RPM, even if installed to an alternative location.

I'm not sure it's prudent to start going down the rabbit hole of allowing arbitrary mixing-and-matching of RPM- and non-RPM-supplied build-time dependencies for TORQUE in the upstream spec file.  That seems to me like a can of worms that will quickly become a support nightmare.

Is there a specific rationale why (1) `torque-*` are being built and installed from RPM, (2) `hwloc` (and associated header/support files) is being installed via something other than RPM, **_and_** (3) support for this is _required_ from the upstream vendor?